provider/bikeshed/src/units/data.rs

```diff
@@ -160,17 +181,13 @@ fn test_basic() {
         .payload;
 
     let units_us = us_locale.get().to_owned();
-    let long = units_us.long.get(&Count::One).unwrap();
+    let long = units_us.patterns.get(&Count::One).unwrap();
     assert_eq!(long, "{0} meter");
-    let short = units_us.short.get(&Count::One).unwrap();
```

Comment:
keep these tests

Reply:
Done for us_locale:
We have splatted now the data and the long, short and narrow is part of the data keys.